man/man1/ec2uploadimg.1

```diff
@@ -247,6 +247,8 @@ argument and using the
 .I --vpc-subnet-id
 argument is incompatible. When targeting multiple regions the subnet-id must
 be set in the configuration file.
+.IP "--enclave-options"
+Enables AWS Nitro Enclave options (Optional)
```

Comment:
The argument is `enclave-options` as such I would expect that as a user I have to provide some options with the argument. If it is just a flag then this has to be renamed. If there are options to be set hen the documentation needs to tell me what those are.

Reply:
Well the man page still states "Enclave options: what are the options? The man page should explain what happens when this feature is used. We should not assume that the user of ec2uploadimg knows the code and knows what happens when the command line switch is used.